collections/_article/release-candidate-godot-4-1-rc-1.md

```diff
@@ -0,0 +1,93 @@
+- GDExtension: Add GDExtension `@since` attribute ([GH-78518](https://github.com/godotengine/godot/pull/78518)).
+- GDScript: Fix a race condition in ScriptServer ([GH-76586](https://github.com/godotengine/godot/pull/76586)).
+- GDScript: Fix errors destroying script with static variables ([GH-78521](https://github.com/godotengine/godot/pull/78521)).
+- GDScript: Fix regression when checking for virtual function implementation ([GH-78533](https://github.com/godotengine/godot/pull/78533)).
+- GUI: Use cached saturation for color picker when value is 0 ([GH-78486](https://github.com/godotengine/godot/pull/78486)).
+- GUI: Fix SVG tag closing for OT font glyphs ([GH-78543](https://github.com/godotengine/godot/pull/78543)).
+- GUI: GraphEdit: Fix port hotzones at zoom levels other than 100% ([GH-78673](https://github.com/godotengine/godot/pull/78673)).
+- Import: Fix exporting MeshInstances without a Skeleton in the GLTF module ([GH-77545](https://github.com/godotengine/godot/pull/77545)).
+- Multiplayer: Do not serialize `MultiplayerSpawner.spawn_function` ([GH-78409](https://github.com/godotengine/godot/pull/78409)).
+- Multiplayer: Fix delta variables index decoding ([GH-78709](https://github.com/godotengine/godot/pull/78709)).
+- Navigation: Fix `NavigationMesh` not clearing old polygons ([GH-78596](https://github.com/godotengine/godot/pull/78596)).
+- Navigation: Fix `NavObjects` map assignments ([GH-78665](https://github.com/godotengine/godot/pull/78665)).
+- Porting: Windows: Fix mouse capture when button up message is missed ([GH-72720](https://github.com/godotengine/godot/pull/72720)).
+- Rendering: Add warnings and fallbacks for particle sub-emitters when using GL Compatibility ([GH-78490](https://github.com/godotengine/godot/pull/78490)).
+- Rendering: Reset filter/repeat state of textures in GL Compatibility when render target is cleared ([GH-78620](https://github.com/godotengine/godot/pull/78620)).
+- Rendering: Use a filter with mipmaps when initializing textures with mipmaps in GL Compatibility renderer ([GH-78720](https://github.com/godotengine/godot/pull/78720)).
+- XR: Fix OpenXR Passthrough mode ([GH-78135](https://github.com/godotengine/godot/pull/78135)).
+- XR: Apply reprojection in multiview for our cluster lookup ([GH-78499](https://github.com/godotengine/godot/pull/78499)).
+- XR: Fix incorrect depth buffer option in OpenXR ([GH-78550](https://github.com/godotengine/godot/pull/78550)).
+- Documentation and translation updates.
+
+A previously included fix has been reverted due to regressions and will be addressed in a future version of Godot:
+
+- Editor: Revert "Fix paste value not updated in dictionaries/arrays" ([GH-78643](https://github.com/godotengine/godot/pull/78643)).
+
+This release is built from commit [`1f9e540f1`](https://github.com/godotengine/godot/commit/1f9e540f14edbf2d496a1421f8d37e5b483c4c53) (see [README](https://downloads.tuxfamily.org/godotengine/4.1/rc1/README.txt)).
```

Comment:
:gdunacceptable: style change! Just kidding, it makes sense :)

Reply:
I copied it from RCs for something else 🤔